README.md

```diff
@@ -1,2 +1,57 @@
-# Sakurabook
-Documentation and guides to Sakurabook Shopify app. 
+# <img src="./img/App%20icon.png" alt="drawing" width="50"/> Welcome to Sakurabook project!
+
+Documentation and guides for Shopify [Sakurabook](https://sakurabook.jp/) app, created by Parkfield.
+
+## Contents
+
+- [Getting Started](./en/getting-started.md)
+- [Setting up the app in Shopify](./en/setting-up-the-app-in-shopify.md)
+- [Usage Guides](./en/usage-guides.md)
+- [Release Notes](./en/release-notes.md)
+- [Links](./en/links.md)
+
+Sakurabook is a plugin for pay-as-you-go reservation service which can be used within the Shopify ecosystem. It can be installed into your Shopify store via [Shopify App Store](https://apps.shopify.com/). It is the first Shopify reservation app that supports the Japanese language. It will fit your business - you can choose if you want to book spaces like rental apartments, studios, or services like therapists, hairdressers, or yoga teachers.
+
+📌 The key benefits of the app are
+
+- **Flexible length of reservations**: Allow customers to make bookings with precision to the minute.
+- **Book spaces or services in Japanese**: The reservation plugin for spaces or services is available in Japanese.
+- **Meet online with the customers via Zoom**: Boost your business online after Covid-19.
+- **Keep track of your bookings in Google Calendar**: See the booking in the connected Google Calendar.
+
+This guide is for the Shopify merchants who are the main users of the app. It'll be useful for you when you need help with the installation and setting up of the Sakurabook app.
```

Comment:
Thinking about specifying what we meant by 'the app' so it is clear, it's the admin part, not the storefront part.

Reply:
Maybe I wouldn't say useful, I would say `Follow this guide for proper installation and settings` or something like that. Because we can guarantee the functionality only if the merchant follows this guide.